students/BrandonHenson/Lesson3/mailroom.py

```diff
@@ -0,0 +1,82 @@
+# Brandon Henson
+# 4/8/18
+# Lesson 03 mailroom.py
+
+donor_history = [
+            ['Brandon Henson', 1005.49, 3116.72, 5200],
+            ['Alicia Henson', 21.47, 1500],
+            ['Michael Green', 2400.54],
+            ['Brandon Henson Jr', 355.42, 579.31],
+            ['Kaiya Henson', 636.9, 850.13, 125.23]
+    ]
+name_list = []
+for i in donor_history:
+    q = str(i[0])
```

Comment:
No need for the call to `str` here. In general it's a good idea to use meaningful variable names, for everything except for counter/index variable and things like that. Since you're going to the trouble of reassigning this, I'd use `name = i[0]` or else you could just do `name_list.append(i[0])`